modules/core/src/test/java/org/apache/ignite/internal/processors/security/cache/CacheOperationPermissionCheckTest.java

```diff
@@ -82,12 +137,50 @@ private void testCrudCachePermissions(boolean isClient) throws Exception {
             c -> c.containsKey("key"),
             c -> c.remove("key"),
             c -> c.removeAll(Collections.singleton("key")),
-            IgniteCache::clear,
             c -> c.replace("key", "value"),
             c -> c.putIfAbsent("key", "value"),
             c -> c.getAndPut("key", "value"),
             c -> c.getAndRemove("key"),
-            c -> c.getAndReplace("key", "value")
+            c -> c.getAndReplace("key", "value"),
+            IgniteCache::clear
         );
     }
+
+    protected void checkOperations(Ignite node) {
+        for (Consumer<IgniteCache<String, String>> c : operations()) {
+            c.accept(node.cache(ALL_PERM_TEST_CACHE));
+            c.accept(node.cache(NEW_CACHE)); // if defaultAllowAll == false, there will be exeption
+
+            assertThrowsWithCause(() -> c.accept(node.cache(CREATE_TEST_CACHE)), SecurityException.class);
+        }
+    }
+
+    protected SecurityPermissionSet getSecurityPermissionSet(SecurityPermission... systemPerms) {
```

Comment:
It's better to inline this method. Then we'll get some code duplication, but test method will be more understandable.